src/messaging/tests/TestReliableMessageProtocol.cpp

```diff
@@ -179,6 +179,9 @@ class MockSessionEstablishmentDelegate : public ExchangeDelegate
 
 void test_os_sleep_ms(uint64_t millisecs)
 {
+#ifdef __MBED__
+    usleep((__useconds_t)(millisecs * 1000));
```

Comment:
Could we just use `usleep` on all our platforms?  It seems to exist on Mac and Linux, at least, and on esp32, and I'm pretty sure those are all the places we currently run out unit tests.

It would be nice to not have ifdefs here if we can avoid them.